packages/nextjs/components/scaffold-eth/RainbowKitCustomConnectButton.tsx

```diff
@@ -35,10 +35,10 @@ export const RainbowKitCustomConnectButton = () => {
               if (chain.unsupported || chain.id !== configuredNetwork.id) {
                 return (
                   <div className="dropdown dropdown-end">
-                    <button tabIndex={0} className="btn btn-error btn-sm dropdown-toggle">
+                    <label tabIndex={0} className="btn btn-error btn-sm dropdown-toggle">
```

Comment:
In your second video, it seems that there is no separation between this label and the faucet button. Not sure if it's because of this change, but could we fix it?

Reply:
Actually, we are not showing `Wrong Network` and `Faucent Button` at the same time, like the `Faucet Button` is [hidden](https://github.com/scaffold-eth/scaffold-eth-2/blob/dd5082c400f07e7ce98879770290e4d6a49ddfa0/packages/nextjs/components/scaffold-eth/FaucetButton.tsx#L36) when you are on the `Wrong Network` / `Not Connected`. I hardcoded the `line 35` in this file to `true` in the above video for testing purposes. 

But found this on `main` branch : 
![Screenshot 2023-05-10 at 1 57 36 AM](https://github.com/scaffold-eth/scaffold-eth-2/assets/80153681/7df750a9-6dd7-485a-8782-8a05d01e2f28)


Added a minimal fix at [07df166](https://github.com/scaffold-eth/scaffold-eth-2/pull/342/commits/07df166c878e606a5009ce07c614949ab49a7ea5) : 

https://github.com/scaffold-eth/scaffold-eth-2/assets/80153681/db4a70d7-1221-4d7f-bb48-9e9b03f50f77

But on too small devices it again merges :(, The way things are structured now leads me to add an [`xs` breakpoint ](https://tailwindcss.com/docs/screens#adding-smaller-breakpoints) to handle smaller devices  (this will also help responsiveness of footer) but not really sure if it's worth adding to `tailwind.config.js` and go that route.